frontend/test/metabase/scenarios/visualizations/pivot_tables.cy.spec.js

```diff
@@ -88,25 +86,27 @@ describe("scenarios > visualizations > pivot tables", () => {
   });
 });
 
+const testQuery = {
+  type: "query",
+  query: {
+    "source-table": ORDERS_ID,
+    aggregation: [["count"]],
+    breakout: [
+      ["fk->", ["field-id", ORDERS.USER_ID], ["field-id", PEOPLE.SOURCE]],
+      [
+        "fk->",
+        ["field-id", ORDERS.PRODUCT_ID],
+        ["field-id", PRODUCTS.CATEGORY],
+      ],
+    ],
+  },
+  database: 1,
+};
+
```

Comment:
I was about to ask how is this different from the API implementation we're already using if we need to type the whole query anyway. But then I realized we're avoiding the whole `cy.request(...)` and this also saves us from the callback hell.
Was that the main idea for implementing this helper?

Reply:
Yeah the main hope is to skip UI setup. I decided to add it when I [saw this setup](https://github.com/metabase/metabase/pull/14164/files#diff-62a6eba56c1b01f8d507a115baa97ce9d45f644d5e3e4b15416ccefc409bb396R46-R66) in your PR from earlier.

Saving a question via the API is about the same as this. It's a bit more of a hassle to save and retrieve the question by id rather than plop it in the URL. I guess it saves two API roundtrips, which could add up across the whole test suite but doesn't matter much for a given test.